lib/ansible/modules/dnf.py

```diff
@@ -1239,13 +1206,8 @@ def ensure(self):
             failure_response['msg'] = "Depsolve Error occurred: {0}".format(to_native(e))
             self.module.fail_json(**failure_response)
         except dnf.exceptions.Error as e:
-            if to_text("already installed") in to_text(e):
-                response['changed'] = False
-                response['results'].append("Package already installed: {0}".format(to_native(e)))
```

Comment:
Oh I see, so I suppose it's the same as for the case from the first comment, so it should be fine.